website/databend/docs/dev/development/tracing.md

```diff
@@ -150,3 +150,24 @@ Jun 10 16:40:36.167 DEBUG ThreadId(309) databend_query::pipelines::transforms::t
 Jun 10 16:40:36.168 DEBUG ThreadId(309) databend_query::pipelines::transforms::transform_expression_executor: (projection executor) execute, actions: [Input(ActionInput { name: "(sum((number + 1)) + 1)", return_type: UInt64 })]
 Jun 10 16:40:36.168 DEBUG ThreadId(309) databend_query::pipelines::transforms::transform_projection: Projection cost: 241.864µs
 ```
+## Distributed tracing with Jaeger
+
+### Start Databend
+```
+QUERY_LOG_LEVEL="DEBUG" ./databend-query
```

Comment:
is it a typo? seems it sould be `LOG_LEVEL="DEBUG" ./databend-query`

Reply:
Fixed, thank you.